File: Accenture/3. Dividend/code.py
def find_dividend_index(A, D, Q, R):
    dividend = D * Q + R
    if dividend in A:
        return A.index(dividend) 
    else:
        return -1 

# Test case
A = [15, 10, 30, 25, 50]
D = 5
Q = 2
R = 5
print(find_dividend_index(A, D, Q, R))  # Output: 0

File: Accenture/3. Dividend/code.cpp
#include <iostream>
#include <vector>
using namespace std;
int findDividendIndex(vector<int>& A, int D, int Q, int R) {
    int dividend = D * Q + R;
    for (int i = 0; i < A.size(); i++) {
        if (A[i] == dividend) {
            return i;
        } }
    return -1;
}
int main() {
    vector<int> A = {15, 10, 30, 25, 50};
    int D = 5, Q = 2, R = 5;
    int result = findDividendIndex(A, D, Q, R);
    cout << result << endl; // Output: 0
    return 0;
}
